index.d.ts

```diff
@@ -42,9 +42,12 @@ hexRgb('#cd2222cc');
 
 hexRgb('#cd2222cc', {format: 'array'});
 //=> [205, 34, 34, 0.8]
+
+hexRgb('#cd2222cc', {format: 'css'});
+//=> 'rgba(205, 34, 34, 0.8)'
 ```
 */
 declare function hexRgb(hex: string): hexRgb.RgbaObject;
-declare function hexRgb(hex: string, options: hexRgb.Options): hexRgb.RgbaTuple;
+declare function hexRgb(hex: string, options: hexRgb.Options): hexRgb.RgbaTuple | string;
```

Comment:
The return value should be specifically typed per `format` option value, so the user gets the correct return value for the specific option.